modules/information-collected-by-telemetry.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * telemetry/about-telemetry.adoc
+:_content-type: CONCEPT
+[id="information-collected-by-telemetry_{context}"]
+= Information collected by Telemetry
+
+Telemetry does not collect identifying information such as user names, passwords, or the names or addresses of user resources.
+
+Telemetry collects the following information:
+
+* API, `roxctl` and user interface (UI) features and settings to know how you use {rh-rhacs-first}, which helps prioritize efforts.
+* The time you spend on UI screens to help us improve user experience.
+* The integrations used to know if there are integrations that you never used.
```

Comment:
**issue:** 

- s/integrations used/integrations are used
- s/you never/you have never